src/implementations/local-paraswap-sdk.ts

```diff
@@ -200,11 +200,12 @@ export class LocalParaswapSDK implements IParaSwapSDK {
       others: [],
       side,
       // For V5 tests, put Augustus V5 address here
-      // contractAddress: '0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57',
-      contractAddress: '',
+      contractAddress: '0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57',
```

Comment:
Can you leave the implementation to be working on V6?

Reply:
done